android/app-example/src/androidTest/java/com/badoo/ribs/example/rib/hello_world/HelloWorldTestDeferred.kt

```diff
@@ -27,7 +29,9 @@ class HelloWorldTestDeferred {
             override fun helloWorldInput(): ObservableSource<HelloWorld.Input> = empty()
             override fun helloWorldOutput(): Consumer<HelloWorld.Output> = Consumer {}
             override fun ribCustomisation(): Directory = AppRibCustomisations
-            override fun activityStarter(): ActivityStarter = ribTestActivity
+            override fun activityStarter(): ActivityStarter = ribTestActivity.activityStarter
```

Comment:
Wasn't that a part of the default interface?

Reply:
Yes, but you still need to satisfy it at least the first time when you create it, right?

Reply:
Right, didn't notice